app.ts

```diff
@@ -1,9 +1,18 @@
-const enviormentConfig = require("./config/environment");
+import { Server } from "http";
+
+const environmentConfig = require("./config/environment");
 const expressConfig = require("./config/express");
 const bootstrap = require("./utils/bootstrap");
 
-(async function () {
-    await enviormentConfig();
-    await expressConfig();
+async function startAgent(): Promise<Server> {
+    await environmentConfig();
```

Comment:
Bad name for a function, should describe an action rather than an item. 'await configureEnvironment()' would be a proper one.